src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
The messge is shown when the dns check fails on a url-field. [ref](https://github.com/symfony/symfony/blob/bfa28d6e41559e2b06c706c12fc6fd6d3643d1b5/src/Symfony/Component/Validator/Constraints/UrlValidator.php#L83) Do you think a normal user understands (or at least have a clue) what the error is if we are using "värddatorn"?